etc/bash_completion.sh

```diff
@@ -1,4 +1,4 @@
 _git_vendor()
 {
-    __gitcomp "add list update"
+    __gitcomp "add list update push"
```

Comment:
i think `upstream` better describes the operation

Reply:
thanks, I will